Origin/sonya

CheshirLvova opened the pull request.

CheshirLvova committed:
Subjects view

closes #30

CheshirLvova committed:
Subjects View (bugs fix)

closes #30

CheshirLvova committed:
Add new views to Subjects pages

CheshirLvova committed:
Merge branch 'master' into origin/Sonya

noteezy commented on src/LECHO.Core/AccountManagement.cs:
```diff
@@ -20,7 +20,9 @@ public class UserNotFoundException : Exception
         public static Users GetUser(string username)
         {
             var user = database.Users.FirstOrDefault(u => u.Login == username);
-            if (user == null) throw new UserNotFoundException("Користувач з таким логіном не знайдений");
```
Таки username == логін, а не ім'я

noteezy commented on src/LECHO.Core/SubjectsAccess.cs:
```diff
@@ -0,0 +1,40 @@
+            var subject = database.Subjects.ToArray().Where(s => s.Semester == id).Select(s => s).ToArray();
+            return subject;
+        }
+        public static Users GetLecturer(int id)
```
Ці функції мають бути в юзер функціях а не subject.

noteezy commented on src/LECHO.Core/SubjectsAccess.cs:
```diff
@@ -0,0 +1,40 @@
+            var user = database.Users.FirstOrDefault(u => u.UserId == id);
+            return user;
+        }
+        public static Students GetStudent(int id)
```
Ці функції мають бути в юзер функціях а не subject.

noteezy commented on src/LECHO.Web/Controllers/SubjectsController.cs:
```diff
@@ -50,8 +46,43 @@ public ViewResult Subjects()
+
+        public ViewResult SubjectsSecondTerm()
+        {
+            var map = new Dictionary<string, string>();
```
замість цього мала б бути функція. цей словник за словами Назара треба забрати.

noteezy commented on src/LECHO.Web/wwwroot/css/site.css:
```diff
@@ -146,47 +146,59 @@ div.profile img {
-.chit {
-    padding-top:23px;
+
+.older-students {
```
дивна назва класу, не зрозуміло що робить

noteezy reviewed (changes requested):
*не гнівне рев'ю, чесно*

CheshirLvova committed:
Add new views to Subjects pages (with changes)

Виправлено синтаксичну помилку у повідомленні однієї з функцій AccountManagment
Прибрано зайвий код
Змінено назву незрозумілого класу
Закрито #41

noteezy reviewed (approved):
зауваження виправлено
